_data/governance.yml

```diff
@@ -0,0 +1,61 @@
+introText: This section is updated every 6 months by the Scala Center.
```

Comment:
I think this comment is not necessary. We should anyway update the content as frequently as necessary (sometimes more than once every six months, but most of the time I guess this will not change within a year).